torch/quantization/make_module.py

```diff
@@ -0,0 +1,50 @@
+import torch
+
+_operation_from_float = {
```

Comment:
if those are the only ops you support - why not just create classes explicitly at initialization instead of lazy creating them on demand? (you can still use for loop iteration). That way at least you'd get working pickle behavior and more hope to make things work with TorchScript.

Also, how many layers to you expect to have here? Many of the torch.nn.functional routines already have hard-coded Module equivalents, e.g. ReLU, Pad, etc.

Reply:
It is crucial to get things working with torchscript, so please move this code to init. The list  of ops that we would need is add, cat, mul.   We can start with add and cat for now.

Reply:
I think this comment is now obsolete?